assets/queries/ansible/aws/kms_key_with_vulnerable_policy/query.rego

```diff
@@ -0,0 +1,35 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  aws_kms := task["community.aws.aws_kms"]
+  policy_exists := object.get(aws_kms, "policy", "undefined") != "undefined"
+
+  statement = aws_kms.policy.Statement[_]
+  check_permission(statement) == true
+   
+    result := {
+            "documentId": document.id,
+            "searchKey": sprintf("name=%s.{{community.aws.aws_kms}}.policy", [task.name]),
+            "issueType": "IncorrectValue",
+            "keyExpectedValue": sprintf("name=%s.{{community.aws.aws_kms}}.policy is correct", [task.name]),
+            "keyActualValue": sprintf("name=%s.{{community.aws.aws_kms}}.policy is incorrect", [task.name]),
```

Comment:
The correctness of the policy here is not clear, could you specify what is or is not expected in the resource?